R/latent_gamma.R

```diff
@@ -0,0 +1,85 @@
+#' Draws from the posterior predictive distribution of the `latent_gamma` family
+#'
+#' See [brms::posterior_predict()].
+#'
+#' @param i The index of the observation to predict
+#' @param prep The result of a call to [brms::posterior_predict()]
+#' @param ... Additional arguments
+#' @family postprocess
+#' @autoglobal
+#' @export
+posterior_predict_latent_gamma <- function(i, prep, ...) { # nolint: object_length_linter
+  mu <- brms::get_dpar(prep, "mu", i = i)
+  shape <- brms::get_dpar(prep, "shape", i = i)
+
+  obs_t <- prep$data$vreal1[i]
+  pwindow_width <- prep$data$vreal2[i]
+  swindow_width <- prep$data$vreal3[i]
+
+  .predict <- function(s) {
+    d_censored <- obs_t + 1
+    # while loop to impose the truncation
+    while (d_censored > obs_t) {
+      p_latent <- runif(1, 0, 1) * pwindow_width
+      d_latent <- rgamma(1, shape = shape[s], rate = shape[s] / mu[s])
```

Comment:
very minor but here you parameterise it in terms of the rate and below in terms of the scale. Could you switch so they all use the same parameterization?

Reply:
Yes agree this is vestage of me thinking there was a bug, will change

Reply:
https://github.com/epinowcast/epidist/pull/273/commits/6c0ef8a76b1e285a37e5a3a3776bb7ded8a780cb